lib/kaiser/kaiserfile.rb

```diff
@@ -91,5 +93,9 @@ def type(value)
 
       @server_type = value
     end
+
+    def service(name, image: name)
+      @services << { name => { image: image } }
```

Comment:
Maybe just me, but array of single-key hashes seems better suited to just be a single `{ name: properties }` hash, unless we want to support duplicate names.

But since Docker would likely complain about duplicate names, we could take advantage of the single hash approach to detect them early.

```suggestion
      raise "duplicate service #{name.inspect}" if @services.key?(name)
      @services[name] = { image: image }
```

Of course, the initializer would have to change a bit.
```ruby
@services ||= Config.kaiserfile.services.map { |name, info| Service.new(envname, name, info) }
```
(or something)

Reply:
Ok that makes sense. Will fix